config/generate/generate.go

```diff
@@ -72,14 +72,15 @@ func SetDefaults(envName string, forceDefault bool, CustomConfig interface{}, No
 
     if len(ConfigUpdate) == 0 {
         //Setting default paths for the config file
-        Defaults.IPTable = defaultPath + "p2p/iptable/ip_table.json"
+        Defaults.IPTable = defaultPath + "p2p/ip_table.json"
```

Comment:
We do not want to overwrite the default IPTable rather spawn. I would recommend a fix where we have different naming convention for the IPTable getting updated.

Reply:
I believe you're currently doing the [opposite](https://github.com/Akilan1999/p2p-rendering-computation/blob/88dddd023022a4ad66154526ed3e0257443fbec6/.gitignore#L13), i.e: the generated iptable is being ignored in the .gitignore.

Reply:
I will merge this. Thanks !